src/Common/src/CoreLib/System/IO/Path.Windows.cs

```diff
@@ -34,7 +41,7 @@ public static string GetFullPath(string path)
                 throw new ArgumentNullException(nameof(path));
 
             // If the path would normalize to string empty, we'll consider it empty
-            if (PathInternal.IsEffectivelyEmpty(path))
+            if (PathInternal.IsEffectivelyEmpty(path.AsSpan()))
```

Comment:
I see lots of cases where you need to "know" to use span overloads.  What can we do (other than code review) to keep ourselves honest here?  Analyzer?

Reply:
Not sure outside of it will fail to build.

Reply:
> it will fail to build

Will it?  I thought we had overloads:
https://github.com/dotnet/corefx/blob/f25eb288a449010574a6e95fe298f3ad880ada1e/src/Common/src/CoreLib/System/IO/PathInternal.Unix.cs#L92-L100

I guess if there aren't overloads my comment does not apply.  I'm talking about the places where there are overloads.  It's not really a comment for this PR though, just a thought.

Reply:
Looks like @tannergooding has suggested the same already: https://github.com/dotnet/corefx/issues/26174.